src/Definition/Entity.php

```diff
@@ -113,6 +119,9 @@ public function setSource(?string $source): self
         return $this;
     }
 
+    /**
+     * @return class-string|null
```

Comment:
Shouldn't it be some `class-string<SourceInterface>|null`?

Reply:
Added ```class-string<SourceInterface>|null```